code/game/machinery/Sleeper.dm

```diff
@@ -282,7 +282,7 @@
 
 /obj/machinery/sleeper/party
 	name = "party pod"
-	desc = "Sleepers were made illegal in the year 2559, as a result of finding doses of lead acetate in every chemical dosage. This is one of the left-overs, that's been retrofitted to a 'party pod'."
+	desc = "Most sleepers were made illegal in the year 2559, after a study found they were the cause of thousands of lead acetate poisonings. This is one of the leftover machines, that's been rebranded into a 'party pod'."

```

Comment:
into implies a change, perhaps as would work?
Yeet me if I'm being pedantic

Reply:
thousands of poisonings

Reply:
I mean it's not great. Still ok tho.
"Lead was discontinued as a plating material after it was found that it was the cause of hundreds of poisonings,"
Just a bit verbose, could use an added reports at the end, or just drop it to lead poisonings if depending on how important the distinction is.
Either way wait till the morning my man.

Reply:
What do you think of:

> “Sleeper” units were once known for their healing properties, until a lengthy investigation revealed they were also dosing patients with deadly lead acetate. This appears to be an old “sleeper” unit repurposed as a “Party Pod”. It’s probably not a good idea to use it.